optaplanner-core/src/main/java/org/optaplanner/core/api/score/comparator/NaturalScoreComparator.java

```diff
@@ -23,7 +23,10 @@
 
 /**
  * Compares by the natural order of {@link Score}s.
+ * 
+ * @deprecated If you need this, it is trivial to implement.
```

Comment:
"it is trivial to implement" says the rocket engineer on how to build a V2 missle :)
Mention mention how to build a comparator on natural comparison? Especially in the upgrade recipe, to save users from have to look that up on SO.

Reply:
Ah, its just Comparable::compareTo apparently?